<launch>
<!-- Compare mesh between two meshes -->
    <arg name="visualization" default="true"/>
    <arg name="path" default="/mnt/dataset/Results/dc_coverage/benchmark_0"/>
    <arg name="trajectory_path" default="/mnt/dataset/Datasets/dc_coverage/0/benchamrk_raw_0.txt"/>

    <node pkg="tf" type="static_transform_publisher" name="world_map" args="0 0 0 0 0 0 world map 10"/>
    <node pkg="photogrammetry_evaluations" type="replay_path_node" name="test_mesh" output="screen" required="true">
        <param name="benchmark_dir_path" value="$(arg path)"/>
        <param name="trajectory_path" value="$(arg trajectory_path)"/>
        <param name="visualize" value="$(arg visualization)"/>
        <param name="replay_time" value="500.0"/>
        <param name="output_path" value="$(arg path)"/>
    </node>

    <node pkg="grid_map_visualization" type="grid_map_visualization" name="grid_map_visualization" output="screen">
        <rosparam command="load" file="$(find adaptive_viewutility)/launch/visualize.yaml" />
    </node>

    <group if="$(arg visualization)">
        <node type="rviz" name="rviz" pkg="rviz" args="-d $(find photogrammetry_evaluations)/launch/config_replay.rviz" required="true"/>
    </group>
</launch>
